nflow-engine/src/main/java/io/nflow/engine/service/WorkflowInstanceService.java

```diff
@@ -61,8 +61,21 @@ public WorkflowInstanceService(WorkflowInstanceDao workflowInstanceDao, Workflow
    * @throws NflowNotFoundException If workflow instance is not found.
    */
   public WorkflowInstance getWorkflowInstance(long id, Set<WorkflowInstanceInclude> includes, Long maxActions) {
+    return getWorkflowInstance(id, includes, maxActions, false);
+  }
+
+  /**
+   * Return the workflow instance matching the given id.
+   * @param id Workflow instance id.
+   * @param includes Set of properties to be loaded.
+   * @param maxActions Maximum number of actions to be loaded.
+   * @param queryArchive Query archive if not found from main tables.
```

Comment:
```suggestion
   * @param queryArchive Query archive tables if not found from main tables.
```
I would do the same change ("archive" -> "archive tables") in all javadocs talking about archive tables and main tables.